tests/vmi_multus_test.go

```diff
@@ -296,7 +296,7 @@ var _ = Describe("[Serial]Multus", func() {
 				tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
 
 				By("Configuring static IP address to ptp interface.")
-				Expect(configInterface(vmiOne, "eth0", "10.1.1.1/24")).To(Succeed())
+				Expect(configInterface(vmiOne, "eth0", "10.1.1.1")).To(Succeed())
```

Comment:
I don't think you can drop the cidr from here.

Reply:
right, good point